deps/rabbit_common/src/file_handle_cache_stats.erl

```diff
@@ -31,20 +31,32 @@ init() ->
 
 update(Op, Bytes, Thunk) ->
     {Time, Res} = timer_tc(Thunk),
-    _ = ets:update_counter(?TABLE, {Op, count}, 1),
-    _ = ets:update_counter(?TABLE, {Op, bytes}, Bytes),
-    _ = ets:update_counter(?TABLE, {Op, time}, Time),
-    Res.
+    case application:get_env(rabbit, collect_file_handle_cache_statistics) of
```

Comment:
Using `application:get_env/2` on any hot code path is an absolute no-no because all of those requests go to a single process.